testsuite/classlibrary/TestErrorHandling.sc

```diff
@@ -0,0 +1,33 @@
+
+
+TestThrow : UnitTest {
+
+	test_throw_nested_threads {
+
+		fork { // isolate from surrounding thread
+
+			var call1 = false, call2 = false;
+
+			var r = Routine {
+				var r = Routine {
+					var r = Routine {
+						Error("test").throw;
+					};
+					var next = thisThread;
+					r.exceptionHandler = { call1 = true; next.handleError };
+					r.next;
+				};
+				var next = thisThread;
+				r.next;
+			};
+			var next = thisThread;
+			r.exceptionHandler = { call2 = true; next.handleError };
+			r.next;
+
+			this.assert(call1 & call2, "When throwing an error in a thread, handleError should call parent thread");
```

Comment:
move this assert outside the fork, also think you mean to use && here

Reply:
if it is outside the fork, it won't test the test. But yes, `&&`!

Reply:
you will need to move more than just this line to do it... probably wait on a condition variable